Makefile.dep

```diff
@@ -398,6 +398,15 @@ ifneq (,$(filter stdio_rtt,$(USEMODULE)))
 endif
 
 ifneq (,$(filter shell,$(USEMODULE)))
+  USEMODULE += stdin
+endif
+
+ifneq (,$(filter stdio_ethos,$(USEMODULE)))
```

Comment:
`stdio_ethos` needs `stdio_uart` dependency as it is implemented in it.
It was missing from before too.

If implemented without `stdin` requiring `stdio_uart_rx` would be do the trick.

Reply:
@aabadie "stdio_uart" still missing here (though that is default for all platforms where stdio_ethos would work

Reply:
Added  `stdio_uart` in the list of dependencies.